documentation/onboard-dpg-in-sdkautomation/README.md

```diff
@@ -0,0 +1,37 @@
+# Service Onboard DPG with Swagger CI Pipeline
+
+Service Onboarding DPG with Swagger CI pipeline can help you get generated SDK and ApiView during the procedure of creating swagger PR.
+The benefits are following:
+1. API signature could be reviewed at the same time when we submit a Swagger PR. This could help to detect API issues at the early stage.
+2. It enable service teams to access their SDKs even before their Swagger PR is merged.
+
+This Document describes the process of how to onboard DPG with SDK automation pipeline. __This document is targeting for service team, who wants to onboard DPG.__ 
+
+*If you want to go through details of pipeline implementation, please go to [Integrate DPG and ApiView in SDK Automation Pipeline](Integrate-dpg-and-apiview-in-sdk-automation-pipeline.md).*
+
+## WorkFlow
+
+![workflow-for-service-team](workflow-service-team.png)
+
+__Description:__
+1. Creates a swagger PR with [Guide to design and creation of Data Plane REST API and Client Libraries](https://aka.ms/azsdk/dpcodegen).
+2. If the SDK was generated for this service before, and you don't need to change the autorest configuration in sdk repository, please go to step 4. Otherwise, please go to step 3.
+3. Create branch with name `swagger/<prNumber>` in sdk repository, and add/update autorest configuration file in the branch. Then add a comment `/azp run` in swagger PR.
```

Comment:
I think we still need to work on improving this step. Even if we don't have full workflow created for this yet we want some sort of helper scripts/tools (maybe docker) that can set this up initially and we want to be sure it happens for all the main languages and not just whichever random ones the service team picks.

Reply:
Perhaps another idea is to not have an only have the service teams provide the extra needed properties and the automation handles seeding the new autorest.md creating the branch. 

We could provide some specialized comments in the specs PR that would allow the teams to provided whatever data is needed.

Reply:
please review the latest doc. thx